sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/HttpRetryPolicy.java

```diff
@@ -136,7 +136,7 @@ private Duration getWellKnownRetryDelay(HttpHeaders responseHeaders, int tryCoun
 
     private Response<?> attempt(final HttpRequest httpRequest, final HttpPipelineNextPolicy next, final int tryCount,
         final List<Exception> suppressed) {
-        HttpRequestAccessHelper.setRetryCount(httpRequest, tryCount + 1);
```

Comment:
first try should have 0 count

Reply:
This was a historical inconsistency we should resolve now, where some spot indicate retry count and others try count (which should be retry count + 1)

Reply:
I updated the `HttpRequestAccessHelper.setRetryCount` and friends to be `tryCount` - this way logging and retry policy and coherent in which number they use to describe the same try. 

I believe it does not affect the rest of the logic.